book-store/src/containers/App.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+
+/**
+ * Main Component
+ */
+class App extends React.Component {
+    constructor(){
+        super();
+    };
+
+    render(){
+        return (
+            <div>
```

Comment:
First router will load the app once App component is loaded then we will load header component because it is going to appear in all the pages then header will load the respective components depending on the url.